.rubocop.yml

```diff
@@ -1,17 +1,41 @@
-Rails:
-  Enabled: true
+# Common configuration.
 inherit_from: .rubocop_todo.yml
```

Comment:
this should be on top, before the `Common configuration comment` and line and empty line after it.